Sources/CartonCLI/Commands/Test.swift

```diff
@@ -48,7 +50,8 @@ struct Test: AsyncParsableCommand {
   @Argument(help: "The list of test cases to run in the test suite.")
   var testCases = [String]()
 
-  @Option(help: "Environment used to run the tests, either a browser, or command-line Wasm host.")
+  @Option(
+    help: "Environment used to run the tests, either a browser, node, or command-line Wasm host.")
```

Comment:
I hope string interpolation works in this context. That would help us generate the list of available environments automatically.

```suggestion
  @Option(
    help: "Environment used to run the tests. Available values: \(Environment.allCases.joined(separator: ","))")
```